src/main/model/workoutexercise.ts

```diff
@@ -0,0 +1,32 @@
+'use strict';
+import { Model } from 'sequelize';
+
+export interface WorkoutExerciseAttributes {
+  orderNum: number;
+}
+
+module.exports = (sequelize: any, DataTypes: any) => {
+  class WorkoutExercise
+    extends Model<WorkoutExerciseAttributes>
+    implements WorkoutExerciseAttributes
+  {
+    orderNum!: number;
```

Comment:
This file is necessay. Otherwise adding to join table (WorkoutExercise) with custom attribute won't work.